packages/oas-to-har/src/index.js

```diff
@@ -97,7 +97,21 @@ module.exports = (
     operation = new Operation(oas, operationSchema.path, operationSchema.method, operationSchema);
   }
 
-  const formData = { ...defaultFormDataTypes, ...values };
+  const formData = {
+    ...defaultFormDataTypes,
+    server: {
+      selected: 0,
+      variables: oas.defaultVariables(0),
+    },
+    ...values,
+  };
+
+  // If the incoming `server.variables` is missing variables let's pad it out with defaults.
+  formData.server.variables = {
```

Comment:
nit: should we just move up into the object above or was this done to improve readability?

Reply:
Since I'm loading `values` in as `...values` above I don't have a clean way to re-fill `server.variables` with defaults if they're missing in the block above this.